server/routes/reviews.js

```diff
@@ -1,8 +1,93 @@
-// Product REQs ============================================================ //
-// GETs
+// Reviews REQs ============================================================ //
 const express = require('express');
-let router = express.Router();
+const router = express.Router();
+const axios = require('axios');
+let apiKey = require('../.apiKey.js');
 
-// PUT requests
-// query params not in URI
-// in req.body
+// create axios instance
+let ax = axios.create({
+  baseURL: 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/reviews',
+  timeout: 1000,
+  headers: apiKey,
+});
+
+// middleware applied to all /:product_id URIs
+router.use((req, res, next) => {
+  req.id = req.query.product_id
+  next();
+})
+
+// root ENDpoint route
+router.route('/')
+  .get((req, res) => {
+    req.page = req.query.page || 1;
```

Comment:
Since the heroku API already defaults to these values, this seems redundant. We could try something like 
req.page = req.query.page ? `?page=${req.query.page} : '';
then on line 26, replace ?page=${req.page} with ${req.page}.

This may not be necessary, but it feels odd to write in our own defaults that reflect defaults provided to us.